build/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  build-static-ioc:
+    image: ghcr.io/cnpem/lnls-alpine-3.18-epics-7:v0.11.0
+    volumes:
+      - type: bind
+        source: ../
+        target: /opt/afc-epics-ioc
+    working_dir: /opt/afc-epics-ioc
+    command: bash -c "build/uhal.sh && lnls-build-static-ioc afc-epics-ioc"
+    environment:
+      SKIP_CLEAN: 1
```

Comment:
> Setting SKIP_CLEAN is necessary so the decode-reg executable installed uhal.sh isn't cleaned up before the tarball is generated.

Yet another issue that could probably be resolved by out-of-tree install. :/ My suggestion would be install both in a shared tree, which would be packed into a tarball. Cleaning would then be possible in the IOC source tree if I'm not missing something. But if you are okay with `SKIP_CLEAN`ing by default, let's move on.

Reply:
It would be necessary to test if `make distclean` doesn't also clean the install location.

I'm okay with making this an issue to be fixed later, fair?